src/domain/community/application/application.entity.ts

```diff
@@ -40,34 +38,23 @@ export class Application extends BaseEntity {
   @ManyToOne(
     () => User,
     user => user.focalPoints,
-    { eager: true, cascade: true, onDelete: 'CASCADE' }
+    { eager: true, cascade: true }
   )
   user!: User;
 
   @Field(() => [Question])
   @ManyToMany(
     () => NVP,
     nvp => nvp.id,
-    { eager: true, cascade: true, onDelete: 'CASCADE' }
+    { eager: true, cascade: true }
   )
   @JoinTable({ name: 'application_questions' })
   questions?: Question[];
 
-  @ManyToMany(
-    () => Ecoverse,
-    ecoverse => ecoverse.applications
-  )
-  ecoverse?: Ecoverse[];
-
-  @ManyToMany(
-    () => Challenge,
-    challenge => challenge.applications
-  )
-  challenge?: Challenge[];
-
-  @ManyToMany(
-    () => Opportunity,
-    opportunity => opportunity.applications
+  @ManyToOne(
+    () => Community,
+    community => community.applications,
+    { eager: false, cascade: false, onDelete: 'CASCADE' }
```

Comment:
That's extemely inconsistent, somewhere cascade:true, somewhere false, somewhere onDelete: 'CASCADE', somewhere not. What is the reason? If it is implemented like that, the services will be inconsistent as well and overall the approach to deleting will be extremely inconsistent.